File: easy/Unique-Elements/Unique Elements.cs
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            // do something with line
            ShowUnique(line);
        }
    }
    
    public static void ShowUnique(string line){
        if (line.Length>0){
            string[] nums = line.Split(',');
            string result = "";
            string numStr = "";
            int i = 0;
            while(i<nums.Length){
                if(!nums[i].Equals(numStr)){
                    numStr = nums[i];
                    result += numStr + ",";
                    i++;
                } else i++;
            }
            System.Console.WriteLine(result.Substring(0,result.Length-1));
        }
    }
}

File: easy/Unique-Elements/Unique Elements.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShowUnique(line);
        }
    }
    
    public static void ShowUnique(String line){
        if (line.length()>0){
            String[] nums = line.split(",");
            String result = "";
            String numStr = "";
            int i = 0;
            while(i<nums.length){
                if(!nums[i].equals(numStr)){
                    numStr = nums[i];
                    result += numStr + ",";
                    i++;
                } else i++;
            }
            System.out.println(result.substring(0,result.length()-1));
        }
    }
}
